apps/src/templates/progress/DetailProgressTable.jsx

```diff
@@ -10,7 +10,8 @@ import {levelType, lessonType} from './progressTypes';
 export default class DetailProgressTable extends React.Component {
   static propTypes = {
     lessons: PropTypes.arrayOf(lessonType).isRequired,
-    levelsByLesson: PropTypes.arrayOf(PropTypes.arrayOf(levelType)).isRequired
+    levelsByLesson: PropTypes.arrayOf(PropTypes.arrayOf(levelWithProgressType))
```

Comment:
(rant, no action needed) Totally off-topic, but our 80-character line limit really bums me out sometimes.  Splitting this on two lines is less readable than on one.  :-(

Reply:
i know we disagree about this, but i'm of the opinion that if our lines are getting too long, that probably means something needs to be cleaned up. my attempt to demonstrate this spiraled a little out of control (see #40368), but these PropTypes are much cleaner (and short enough to not need any wrapping) after that change!

Reply:
Oh, I don't disagree with that statement at all, but I do think that 80 is a little short and sometimes encourages shorter, less descriptive names (even when code is well-factored).  If it were up to me, I would probably just disable the line length rules and leave the line breaks up to the discretion of the author.